test/integration/patients/patient/sidebar.js

```diff
@@ -145,4 +148,263 @@ context('patient sidebar', function() {
       .get('.patient-sidebar')
       .should('not.contain', 'Another Group');
   });
+
+  context('engagement status', function() {
+    specify('Active', function() {
+      cy
+        .server()
+        .routePatientActions()
+        .routePatient(fx => {
+          fx.data.id = '1';
+
+          return fx;
+        })
+        .routePatientEngagementStatus(fx => {
+          fx.data.status = 'active';
+          return fx;
+        })
+        .routePatientFlows()
+        .routePrograms()
+        .routeAllProgramActions()
+        .routeAllProgramFlows()
+        .visit('/patient/dashboard/1')
+        .wait('@routePatient');
+
+      cy
+        .get('.patient__sidebar')
+        .find('.patient-sidebar__heading')
+        .contains('Engagement Status')
+        .next()
+        .should('contain', 'Loading...');
+
+      cy
+        .wait('@routePatientEngagementStatus');
+
+      cy
+        .get('.patient__sidebar')
+        .find('.active')
+        .parent()
+        .should('contain', 'Active');
+    });
+
+    specify('Not Scheduled', function() {
```

Comment:
I don't think we need to test all of the statuses.. if there was more logic than just picking a key out of the collection, but just testing a single engagement value for now should be enough